src/testIntegration/java/uk/gov/justice/digital/delius/controller/api/DeliusOffenderAPITest.java

```diff
@@ -44,14 +54,13 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
-import static uk.gov.justice.digital.delius.util.OffenderHelper.anOffender;
+import static uk.gov.justice.digital.delius.OffenderHelper.anOffender;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, properties = {"offender.ids.pagesize=5"})
 @RunWith(SpringJUnit4ClassRunner.class)
 public class DeliusOffenderAPITest {
     @Rule
-    public WireMockRule wireMock = new WireMockRule(wireMockConfig().port(8088).jettyStopTimeout(10000L));
-
+    public WireMockRule wireMock = new WireMockRule(wireMockConfig().port(8088).usingFilesUnderDirectory("src/testIntegration/resources").jettyStopTimeout(10000L));
```

Comment:
A genuine change here - by default Wiremock gets its mappings from `src/test/resources`

Reply:
So could we left the mapping files where they were, or do you think that was too confusing?

Reply:
I think it's not great that Wiremock hardcodes that mappings directory instead of looking on the classpath, so that confused me.  We could have left them where they were... doesn't feel good though.  I'll look a bit harder at using the classpath rather than trumping hardcoding with more hardcoding.